static/src/stylesheets/module/_adslot.scss

```diff
@@ -243,7 +243,6 @@
         position: relative;
         height: auto;
         margin: 0 $gs-gutter/2;
-        width: gs-span(4.5);
```

Comment:
I did testing on multiple browsers and it looks like this is not needed. Although there might be some case where it will do something unexpected. If it will happen we should be able to roll fix out very fast.